### PR TITLE
⛓️ Gatway to Apps name change

### PR DESCRIPTION
**Pull request by WRadoslaw**
- #3583

### CODE REVIEW

File: packages/ui/src/working-groups/types/locks.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const GROUP_TO_LOCK_ID: Record<GroupIdName, LockType> = {
   contentWorkingGroup: 'Content Directory Worker',
   membershipWorkingGroup: 'Membership Worker',
   storageWorkingGroup: 'Storage Worker',
-  gatewayWorkingGroup: 'Gateway Worker',
+  gatewayWorkingGroup: 'Apps Worker',
   distributionWorkingGroup: 'Distribution Worker',
   operationsWorkingGroupGamma: 'Builders Worker',
   operationsWorkingGroupBeta: 'HR Worker',
```
Comment (traumschule) [inline packages/ui/src/working-groups/types/locks.ts:12]: wdyt about
```
  contentWorkingGroup: 'Curator',
  membershipWorkingGroup: 'Membership worker',
  storageWorkingGroup: 'Storage provider',
  gatewayWorkingGroup: 'App provider,
  distributionWorkingGroup: 'Distribution provider',
  operationsWorkingGroupAlpha: 'Builder',
  operationsWorkingGroupBeta: 'HR worker',
  operationsWorkingGroupGamma: 'Marketer',
```

File: packages/ui/src/accounts/types/balances.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const WorkerLocks = [
   'Content Directory Worker',
   'Forum Worker',
   'Membership Worker',
-  'Gateway Worker',
+  'Apps Worker',
   'Builders Worker',
   'HR Worker',
   'Marketing Worker',
```
Comment (traumschule) [inline packages/ui/src/accounts/types/balances.ts:20]: Looks like a double structure with `locks.ts`. Does it make sense to merge them into `app/pages/WorkingGroups/WorkingGroups.tsx`? Note line 62:
>     name: 'Gateway',

**Review by traumschule — COMMENTED** [review]
lgtm except below. Wonder if it makes sense to rename `GatewayWorkerIcon` too later?

For `common/helpers.ts` this may change if accepted: https://discord.com/channels/811216481340751934/1038170229555089458/1045690789071683684
>        tooltipLink: 'https://joystream.gitbook.io/testnet-workspace/system/gateways',

File: packages/ui/src/accounts/types/balances.ts (hunk 2 above)
Comment (WRadoslaw) [reply packages/ui/src/accounts/types/balances.ts:20]: This name is getting mapped by the helper to the Apps, so no worries.

File: packages/ui/src/working-groups/types/locks.ts (hunk 1 above)
Comment (WRadoslaw) [reply packages/ui/src/working-groups/types/locks.ts:12]: @dmtrjsg

Comment (traumschule) [reply packages/ui/src/working-groups/types/locks.ts:12]: with :heart: "unassign me" #3268 (ref #3583)

**Pull request merged**